src/service/gameService.js

```diff
@@ -1,6 +1,8 @@
 const { PREFLOP, FLOP, TURN, RIVER, phases, DECK } = require('../constants')
 const { distributeChipsToWinners, getHandRanks } = require('./winnerService')
 const { encryptCard, decryptHand } = require('./encryptionService')
+const { Room } = require('../models/Room')
+const func = require('joi/lib/types/func')
```

Comment:
Is this import used anywhere?

Reply:
line 4 - yes while writing data to db, func isn't used so removed it